run-tests-jenkins

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-coverage run manage.py test --with-xunit $1
+coverage run manage.py test --verbosity=3 --nomigrations --with-fixture-bundling --with-xunit $1
```

Comment:
Do we now need multiple scripts here? One for running nightlies and one for running commit tests?